src/lib/dnssd/Types.h

```diff
@@ -295,10 +299,13 @@ struct ResolvedNodeData
     }
 };
 
+using DiscoveredNodeData = Variant<CommissionNodeData>;
+
+#if 0
```

Comment:
Please don't leave the `#if 0` bits here.  They should just go away, right?

Reply:
Sorry, missed this somehow during my clean up. Will remove it.